deeplearning4j-nn/src/main/java/org/deeplearning4j/eval/RegressionEvaluation.java

```diff
@@ -107,6 +107,11 @@ public void eval(INDArray labels, INDArray predictions) {
         //https://en.wikipedia.org/wiki/Pearson_product-moment_correlation_coefficient#For_a_sample
         //Doing online calculation of means, sum of squares, etc.
 
+        if (columnNames.size() != labels.size(1) || columnNames.size() != predictions.size(1)) {
+            throw new IllegalArgumentException(
```

Comment:
Maybe change this to -1? @AlexDBlack how are we handling time series? Are we assuming this is multi dimensional or are we going to add an evalTimeSeries similar to classification?